src/algebra/continued_fractions/translations.lean

```diff
@@ -15,6 +15,8 @@ Some simple translation lemmas between the different functions defined in
 
 namespace generalized_continued_fraction
 open generalized_continued_fraction as gcf
+
+section general
```

Comment:
Adding sections like that is nice, but what is even nicer is to add headers for the doc like
```
/-!
### General properties

In this paragraph, we prove ...
-/